docker_image_cleaner/cleaner.py

```diff
@@ -184,72 +150,49 @@ def main():
             # Do nothing! We have enough space
             pass
         else:
-            images = get_docker_images(client)
+            images = client.images.list(all=True)
             if not images:
-                logging.info(f"No images to delete")
+                logging.info("No images to delete")
                 time.sleep(interval)
                 continue
             else:
                 logging.info(f"{len(images)} images available to prune")
 
-            start = time.perf_counter()
-            images_before = len(images)
-
-            if node:
-                cordon(kube, node)
-
-            deleted = 0
-
-            while images and get_used(path_to_check) > gc_low:
-                # Ensure the node is still cordoned
+            for kind in ("containers", "images"):
+                # Ensure the node is cordoned while we prune
                 if node:
                     cordon(kube, node)
-                # Remove biggest image
-                image = images.pop(0)
-                if image.tags:
-                    # does it have a name, e.g. jupyter/base-notebook:12345
-                    name = image.tags[0]
-                else:
-                    # no name, use id
-                    name = image.id
-                gb = image.attrs["Size"] / (2**30)
-                logging.info(f"Removing {name} (size={gb:.2f}GB)")
+                key = f"{kind.title()}Deleted"
+                tic = time.perf_counter()
+                collection = getattr(client, kind)  # client.containers
                 try:
-                    client.images.remove(image=image.id, force=True)
-                    logging.info(f"Removed {name}")
-                    # Delay between deletions.
-                    # A sleep here avoids monopolizing the Docker API with deletions.
-                    time.sleep(delay)
-                except docker.errors.APIError as e:
-                    if e.status_code == 409:
-                        # This means the image can not be removed right now
-                        logging.info(f"Failed to remove {name}, skipping this image")
-                        logging.info(str(e))
-                    elif e.status_code == 404:
-                        logging.info(f"{name} not found, probably already deleted")
-                    else:
-                        if node:
-                            # uncordon before giving up
-                            uncordon(kube, node)
-                        raise
+                    pruned = collection.prune()
                 except requests.exceptions.ReadTimeout:
-                    logging.warning(f"Timeout removing {name}")
+                    logging.warning(f"Timeout pruning {kind}s")
```

Comment:
```suggestion
                    logging.warning(f"Timeout pruning {kind}")
```

I think `kind` already contains the plural